aviary/modules/assembly/envs/spades.yaml

```diff
@@ -3,7 +3,7 @@ channels:
   - bioconda
 dependencies:
   - python <= 3.9
-  - spades = 3.15.4
+  - spades = 4.0.0
```

Comment:
Is it as simple as a drop in replacement like this? I would assume that the python version may need to be revisited as well, it may need to bump up to 3.11 or 3.12.

Reply:
good q - is there a reason python is even specified? Shouldn't that just be handled as a dep of spades?

Reply:
Keeping it pinned below a certain version makes installation a lot more consistent for a longer time. Easier to pinpoint an issue when someone has an error